addon/components/power-calendar-multiple/days.js

```diff
@@ -1,9 +1,18 @@
 import DaysComponent from '../power-calendar/days';
+import fallbackIfUndefined from '../../utils/computed-fallback-if-undefined';
 
 export default DaysComponent.extend({
+  maxLength: fallbackIfUndefined(Infinity),
+
   // Methods
   dayIsSelected(dayMoment, calendar = this.get('calendar')) {
     let selected = calendar.selected || [];
     return selected.some((d) => dayMoment.isSame(d, 'day'));
+  },
+
+  dayIsDisabled(dayMoment) {
+    let numSelected = this.get('calendar.selected.length') || 0;
+    let maxLength = this.get('maxLength') || Infinity;
```

Comment:
Well, the default only handles cases where the maxLength is undefined, but if for some reason is null (which is possible, I could see a user setting the value to null and expect it to not apply)